tests/test_ssds.py

```diff
@@ -112,6 +88,31 @@ def test_upload_name_collisions(self):
                 for ssds_key in StagingS3.upload(root, submission_id):
                     print(StagingS3.compose_blobstore_url(ssds_key))
 
+    def _prepare_local_submission_dir(self, dirname: str, single_file=False) -> str:
```

Comment:
If there were any more than three, I'd suggest looping over the dirs since there's some duplicate code in here, but it's probably fine.